contracts/Inbox.sol

```diff
@@ -94,77 +89,23 @@ contract Inbox is IInbox, Eco7683DestinationSettler, Ownable, Semver {
      * @param _route The route of the intent
      * @param _rewardHash The hash of the reward
      * @param _claimant The address that will receive the reward on the source chain
      * @param _expectedHash The hash of the intent as created on the source chain
-     * @param _prover The address of the hyperprover on the source chain
-     * @return Array of execution results from each call
+     * @param _localProver Address of prover on the destination chain
+     * @param _sourceChainProver Address of prover on the source chain
      */
-    function fulfillHyperInstant(
+    function fulfillMessageBridge(
         Route memory _route,
         bytes32 _rewardHash,
         address _claimant,
         bytes32 _expectedHash,
-        address _prover
-    ) external payable returns (bytes[] memory) {
-        return
-            fulfillHyperInstantWithRelayer(
-                _route,
-                _rewardHash,
-                _claimant,
-                _expectedHash,
-                _prover,
-                bytes(""),
-                address(0)
-            );
-    }
-
-    /**
-     * @notice Fulfills an intent to be proven immediately via Hyperlane's mailbox with relayer support
-     * @dev More expensive but faster than hyperbatched. Requires fee for Hyperlane infrastructure
-     * @param _route The route of the intent
-     * @param _rewardHash The hash of the reward
-     * @param _claimant The address that will receive the reward on the source chain
-     * @param _expectedHash The hash of the intent as created on the source chain
-     * @param _prover The address of the hyperprover on the source chain
-     * @param _metadata Metadata for postDispatchHook (empty bytes if not applicable)
-     * @param _postDispatchHook Address of postDispatchHook (zero address if not applicable)
-     * @return Array of execution results from each call
-     */
-    function fulfillHyperInstantWithRelayer(
-        Route memory _route,
-        bytes32 _rewardHash,
-        address _claimant,
-        bytes32 _expectedHash,
-        address _prover,
-        bytes memory _metadata,
-        address _postDispatchHook
-    )
-        public
-        payable
-        override(IInbox, Eco7683DestinationSettler)
-        returns (bytes[] memory)
-    {
-        bytes32[] memory hashes = new bytes32[](1);
-        address[] memory claimants = new address[](1);
-        hashes[0] = _expectedHash;
-        claimants[0] = _claimant;
-
-        bytes memory messageBody = abi.encode(hashes, claimants);
-        bytes32 _prover32 = _prover.addressToBytes32();
-
-        emit HyperInstantFulfillment(_expectedHash, _route.source, _claimant);
-
-        uint256 fee = fetchFee(
-            _route.source,
-            _prover32,
-            messageBody,
-            _metadata,
-            _postDispatchHook
-        );
+        address _localProver,
+        address _sourceChainProver,
+        bytes calldata _data
```

Comment:
Can we rename `_data` to a more verbose name? Probably something like `proverContext` or `messageSenderData`.